cmd/protoc-gen-connect-go/main.go

```diff
@@ -212,8 +212,7 @@ func generateServiceNameConstants(g *protogen.GeneratedFile, services []*protoge
 }
 
 func generateServiceNameVariables(g *protogen.GeneratedFile, file *protogen.File) {
-	wrapComments(g, "These variables are the protoreflect.Descriptor objects for the ", file.Desc.Name(),
-		" service's methods.")
+	wrapComments(g, "These variables are the protoreflect.Descriptor objects for the RPCs defined in this package.")
```

Comment:
nit: just the RPCs defined in this file. A package can have multiple protos with separate services.
```suggestion
	wrapComments(g, "These variables are the protoreflect.Descriptor objects for the RPCs defined in this file.")
```

Reply:
Oh, never mind. I see you've just mirrored the comments for the service and procedure names.